admin/schedule-form-excludes.php

```diff
@@ -154,40 +155,32 @@
 						$root = new \SplFileInfo( Path::get_root() );
 
 						$size = $site_size->filesize( $root );
-
-						if ( false !== $size ) {
-
-							$size = size_format( $size );
-
-							if ( ! $size ) {
-								$size = '0 B';
-							} ?>
+						$excluded_size = $excluded_site_size->filesize( $root ); ?>
 
 							<code>
 
-								<?php echo esc_html( $size ); ?>
+								<?php
+								echo ( preg_replace( '/[0-9]+/', '', size_format( $size ) ) === preg_replace( '/[0-9]+/', '', size_format( $excluded_size ) ) ) ? esc_html( (int) size_format( $excluded_size ) ) : esc_html( size_format( $excluded_size ) );
```

Comment:
@pdewouters good point re. translation, I'll add that and try to improve the code readability